modules/Products/actions/RelationAjax.php

```diff
@@ -6,47 +6,20 @@
  * The Initial Developer of the Original Code is vtiger.
  * Portions created by vtiger are Copyright (C) vtiger.
  * All Rights Reserved.
+ * Contributor(s): YetiForce Sp. z o.o.
  * *********************************************************************************** */
 
 class Products_RelationAjax_Action extends Vtiger_RelationAjax_Action
 {
 
+	/**
+	 * Constructor
+	 */
 	public function __construct()
 	{
 		parent::__construct();
 		$this->exposeMethod('addListPrice');
 	}
-	/*
-	 * Function to add relation for specified source record id and related record id list
```

Comment:
?

Reply:
because this function is useless  and did not work either.
https://github.com/YetiForceCompany/YetiForceCRM/blob/e624ac7a6275ea030eef3e4a019161650dc0a4fd/modules/Products/actions/RelationAjax.php#L37